src/hotspot/share/prims/jvmti.xml

```diff
@@ -1861,10 +1861,10 @@ jvmtiEnv *jvmti;
         permanent features of the Java platform.</i>
         <p/>
         Resume all virtual threads except those already resumed.
-        Virtual threads may be suspended with
-        <functionlink id="SuspendAllVirtualThreads"></functionlink> or
-        <functionlink id="SuspendThreadList"></functionlink> or
-        <functionlink id="SuspendThread"></functionlink>.
+        Virtual threads may be resumed with
+        <functionlink id="ResumeAllVirtualThreads"></functionlink> or
+        <functionlink id="ResumeThreadList"></functionlink> or
+        <functionlink id="ResumeThread"></functionlink>.
         Virtual threads that are currently resumed do not change state.
       </description>
```

Comment:
This is not right.
The JVM TI Suspend functions are listed intentionally. Similarly, the JVM TI Resume functions are listed in SuspendAllVirtualThreads:
http://100.110.26.5:8080/view/loom/job/loom-fibers-branch-build/lastSuccessfulBuild/artifact/loom/build/linux-x64/images/docs/specs/jvmti.html#SuspendAllVirtualThreads

Reply:
I have reverted this change. Sorry for the noise.